### DISCUSSION TITLE
What I think of your licence...

### DISCUSSION THREAD

**Discussion by Fabrice3D** [airsdk/Adobe-Runtime-Support#890] [General]
As an intro, i'm a dev doing realtime 3d for a living, I have published and shared for free air apps to help produce 3D content since 2006 and used AIR/as3 for that. Been part of open source project like Away3D. There is, at least for me as a freelancer, zero demand for AIR or AS3 in general anymore, for years now, and there are other options to produce 3d work on mobile or in browser.
So basically I use AIR only for own software that I share at times with friends, colleagues.

I'm also an old monkey. I have this now days weird way of thinking: I want to own the stuff I pay for! My car, my house, tools etc... I do not rent, I do not lease, I do not subscribe. I pay for what I need or want. Once!
Adobe lost me as a customer forever for that reason while I used to pay on regular fashion for new versions of some of their softs and your licence will now force me to migrate all my work because I don't want to see your logo on my app. You did NOTHING on or for my apps, the SDK's I have used is 32 or lower. So only the Adobe logo should be displayed. The second thing that is bugging me is that this splash screen is badly implemented, it creates a huge hesitation, delays and the start up isn't smooth at all. The first impression you get from my app right away isn't reflecting at all the care I've placed into it.
All together, this splash screen is a very cheap move.

While if there was a realistic option to pay for a given version of the SDK for my needs, I would certainly buy it if it offers something new that I'd like to use. Once. Gladly even, as I would then also help to keep this software alive.

Instead with this licence, out of principle, I will stop using your great software, just like many new young devs will ignore it.

Now you know what I think of your licence.

**Comment by ajwfrost** [comment]
Thanks for taking the time to provide your feedback!

I can appreciate your position and would say that the original pricing goal we had was that folk would only need to get a license if they were using the new functionality. So, if you are just using SDK version 32, you can continue to use that: Adobe's licenses were perpetual so if you don't need the functionality that we've worked on, you can just avoid using the SDK v33.

We've looked at the splash screen a few times, yes it's not ideal but for people who are hobbyists or charity/low-income, it seemed an appropriate compromise.

The idea of a one-off payment for a version of the SDK is probably not the best option for mobile development where Apple/Google require updates to apps to conform to new requirements every year, and similarly on macOS. On Windows: yes, best to just stick with AIR v32 if you don't need any of the updates/fixes that we've made. Not renting, or taking out a mortgage, or paying a subscription for TV or phone services etc, is a great principle but out of the reach of most of us I'm afraid!

Ultimately: what we're trying to do is keep the AIR platform alive - and progressing - both for the smaller developers and hobbyists, as well as the big businesses and games developers. Given we don't get income from authoring tools like Animate, this is the way we have to go...

thanks

**Reply by Fabrice3D** [reply]
I sincerely hope you will reach your goal. Always liked flash/air/as123 since my first macromedia install back in early nineties. Hope you one day reconsider changing or adding more realistic options for those using the api to distribute free software made with care and countless hours of work. Using the SDK just a few times a year.
Good luck!
